frontend/pages/ManageControlsPage/MacOSSettings/cards/CustomSettings/CustomSettings.tsx

```diff
@@ -20,15 +21,13 @@ import ProfileListHeading from "./components/ProfileListHeading";
 const baseClass = "custom-settings";
 
 interface ICustomSettingsProps {
-  profiles: IMdmProfile[];
-  onProfileUpload: () => void;
-  onProfileDelete: () => void;
+  currentTeamId: number;
+  onDataChange: () => void;
 }
 
 const CustomSettings = ({
-  profiles,
-  onProfileUpload,
-  onProfileDelete,
+  currentTeamId,
+  onDataChange,
 }: ICustomSettingsProps) => {
   const { renderFlash } = useContext(NotificationContext);
   const { currentTeam } = useContext(AppContext);
```

Comment:
I think this should be removed in favor of the currentTeamId from props

Reply:
Agreed. I'll cleanup